vuepress/docs/next/tutorials/backend-developers/generate-microservices-and-micro-frontends.md

```diff
@@ -1,46 +1,45 @@
-# Objective
-
-This tutorial will take you through the steps to install and run the Entando Component Generator powered by JHipster. At the end of the tutorial you can continue your journey by running your components locally or creating an Entando Bundle for the Entando Component Repository.
-
-## Prerequisites
-
--   Java 11
--   Maven 3.0.5+
--   npm 6+
--   git
--   Docker
--   node 12+ (or LTS)
--   A Docker Hub account or access to a docker registry
+# Generate Microservices and Micro Frontends
 
 ## Overview
 
-In this tutorial and in the subsequent tutorials linked at the bottom you'll learn to use the Entando Component Generator powered by JHipster to create and deploy microservices and micro frontends to the [Entando Component Repository](../../docs/ecr/ecr-overview.md) and Entando apps.
+This tutorial shows you how to use the Entando Component Generator powered by [JHipster](https://www.jhipster.tech/) to create microservices and micro frontends for eventual deployment to the [Entando Component Repository](../../docs/ecr/ecr-overview.md) and Entando applications.
 
 ![Entando Component Generator](./img/component-gen-flow.png)
 
 The general flow of the component generation is:
 
-1. Run the Entando blueprint to create your components (Spring Boot microservice and optionally React micro frontends)
+1. Run the Entando Blueprint to create your components (Spring Boot microservice and optionally React micro frontends)
 2. Customize and enhance your generated code
-3. Build a bundle from your components
-4. Deploy to the Entando Component Repository
-5. Install your bundle in your Entando apps
-
+3. Build an Entando Bundle from your components
+4. Deploy to the Entando Component Repository in Kubernetes
+5. Install your Entando Bundle into your Entando application(s)
 
 ## Installation
+The `ent-jhipster` tool is the recommended way to generate microservices and micro frontends but you can also perform these tasks manually. Either way you can use the [Entando CLI](../../docs/reference/entando-cli.md#check-environment) tool to verify you have the necessary prerequisites in place (e.g. Java, npm, git, etc.). 
+
+### CLI Installation
+No additional steps are needed here. The `ent-check-env` tool automatically installs the correct version of JHipster and the Entando Blueprint for you. The `ent-jhipster` tool makes use of those dependencies. 
 
+### Manual Installation
 1. Install JHipster `npm install -g generator-jhipster@6.9.1`
```

Comment:
ent-jhipster is automatically installed by ENT
furthermore with ENT you are sure to install the jhipster version compatible with the Entando version you are targeting

The installation done via

`npm install -g generator-jhipster-entando@6.2.0`

Will just be ignored by ent-jhipster

Reply:
Yeah, but these are the manual steps which I left alone in case someone has issues with ent-check-env or chooses to jump straight to it. Partly historical but also partly showing what's happening under the hood.